test-data/unit/README.md

```diff
@@ -159,7 +159,7 @@ To run mypy on itself:
 
 To run the linter:
 
-    ruff .
+    python3 runtests.py lint
```

Comment:
It reads a bit odd when this command uses `runtests.py`, and the very next line says "You can also run all of the above tests using `runtests.py`". It may be good to mention the underlying lint command here. FWIW, I always run `pre-commit` directly, and only just learned that the `runtests.py` invocation exists. 
```suggestion
    pre-commit run -a
```

Reply:
I prefer to run `lint` using `runtests.py`, since it lets me perform several tasks via a single command, such as `python3 runtests.py lint self pytest-fast`. If we will remove black and ruff from test requirements, we should probably change `runtests.py lint` to run `pre-commit run -a` in order to avoid breaking existing workflows.

Reply:
> we should probably change `runtests.py lint` to run `pre-commit run -a` in order to avoid breaking existing workflows.

It looks like that's already the case :-)
https://github.com/python/mypy/blob/54a2c6d38800195091f23e98e7cd3ef3a6a66702/runtests.py#L68-L69

Reply:
Nice, forgot about it!